app/containers/GamePage/index.js

```diff
@@ -118,7 +117,7 @@ export default class GamePage extends Component {
         if (participants[i].key === data.key) {
           participants.splice(i, 1);
           const num = participants.length;
-          this.setState({ participants, players: num });
+          this.setState({ participants, totalPlayers: num });
```

Comment:
Also you should be doing this.setState({ participants, totalPlayers: num, assign: num >= 3 }) instead of using an if /else loop for setting the state for assign separately